UnityProject/Assets/Scripts/Objects/VisibleBehaviour.cs

```diff
@@ -57,6 +60,16 @@ public override void OnStartClient()
 		{
 			isPlayer = true;
 		}
+
+		ignoredSpriteRenderers = new SpriteRenderer[ignoreSpriteUpdates.Length];
+		for (int i = 0; i < ignoreSpriteUpdates.Length; i++)
+		{
+			SpriteRenderer sr = ignoreSpriteUpdates[i].GetComponent<SpriteRenderer>();
+			if (sr != null)
+			{
+				ignoredSpriteRenderers[i] = sr;
+			}
+		}

```

Comment:
You wouldn't need this loop if you make ignoredSpriteRenders public and just drag them in